airflow/api_connexion/endpoints/config_endpoint.py

```diff
@@ -123,7 +123,7 @@ def get_value(section: str, option: str) -> Response:
                 "Config not found.", detail=f"The option [{section}/{option}] is not found in config."
             )
 
-        if (section, option) in conf.sensitive_config_values:
+        if (section.lower(), option.lower()) in conf.sensitive_config_values:
```

Comment:
Instead of this, I think we should do what we do in the webserver https://github.com/apache/airflow/blob/f349fda125c2251ac4129c2c28fbf6f7dbb69294/airflow/www/views.py#L3840-L3845.

Reply:
I cannot fix the problem, because it checks if the value exists in the configuration as it is. But since we don't read any configuration from the user in this view, it's safe because all the configuration in both side (sensitive dict and configuration dict) are in low format.

Reply:
Agree with @hussein-awala . I think the proposed way is right. The "www/views.py" code displays all the sections/options from the config and then then from sensitive_config_values. What we are preventing here is that differently cased section/option is passed by the user, so the source of section/key are not the config itself, but external to it. We can safely assume that the config has the right sections/keys (that's why the code in views.py works).  But we cannot make the same assumption about the values passed by the user so we have to lowercase them.